### ISSUE TITLE
Choose if bots can enter in game

### ISSUE THREAD

**Issue by olimexsmart** [bstnfrmry/hanabi#216]
At the moment  if the game is inactive, bots automatically are added. This is a problem because maybe you are just waiting for a friend. Moreover, having no admin (which I think is fine) means that the bots can't be kicked out, forcing to create a new game when this happens.

**Comment by bstnfrmry** [comment]
Hey @olimexsmart 

This should definitely not be the case, so it might be a bug.

The only time when bots automatically join the game is during the tutorial (they're call Jane & Adam). Is that what happened to you?

**Comment by olimexsmart** [comment]
Interesting, it's possible. Maybe while waiting someone gets bored and clicks on tutorial, but we never made the association of bots and tutorial. 

With that being the case, maybe it's just the matter of communicating that tutorial means that bots are added, or having the possibility to kick the bots out.

**Comment by bstnfrmry** [comment]
Yes, indeed, I should definitely clarify that, I understand how it can be confusing 👍 

Technically it's not the same game. We create a new game for bots, but we should clearly display an option to go back to the original game at any time.
